### PR TITLE
Create pytest version of control plane ACL test

### PR DESCRIPTION
**Pull request by jleveque**
Create pytest version of control plane ACL test.

### CODE REVIEW

File: tests/cacl/test_control_plane_acl.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+]
+
+SONIC_SSH_PORT  = 22
+SONIC_SSH_REGEX = 'OpenSSH_[\\w\\.]+ Debian'
+
+
+def test_control_plane_acls(duthost, testbed_devices, creds):
+    """Test control plane ACL functionality on a SONiC device
+    """
+    localhost = testbed_devices['localhost']
```
Comment (lguohan) [inline tests/cacl/test_control_plane_acl.py:15]: testbed_devices has been deprecated.

File: tests/cacl/test_control_plane_acl.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                             search_regex=SONIC_SSH_REGEX,
+                             delay=0,
+                             timeout=90)
+
+    if "Timeout when waiting for search string" in res.get('msg', ''):
+        pytest.fail("SSH did not start working when expected")
+
+    # Sleep a while to ensure SSH is stable and prevent timeout error which may
+    # occur when deleting config_service_acls.sh
+    time.sleep(5)
```
Comment (lguohan) [inline tests/cacl/test_control_plane_acl.py:80]: why will the port be blocked again? I suggest to add some more comments here to explain why we have this behavior.

**Comment by lguohan** [comment]
I do not remember if this test was developed in the service acl stage where we can have different implementation of the control plane acl. Now, since we have been using iptables to manage the control plane acl, and you have added many different rules. I would suggest to develop a new test which is comparing the iptables actual and expected output.

people trust iptables implementation nowadays. as long as you have correct iptable rules, we believe it will behave correctly.

File: tests/cacl/test_control_plane_acl.py (hunk 2 above)
Comment (jleveque) [reply tests/cacl/test_control_plane_acl.py]: Removed sleep after further testing, as it is unnecessary here in the Pytest implementation.

File: tests/cacl/test_control_plane_acl.py (hunk 1 above)
Comment (jleveque) [reply tests/cacl/test_control_plane_acl.py]: Thanks. Using new 'localhost' fixture.

**Comment by jleveque** [comment]
> I do not remember if this test was developed in the service acl stage where we can have different implementation of the control plane acl.

Yes, you are correct, it was.

Now, since we have been using iptables to manage the control plane acl, and you have added many different rules. I would suggest to develop a new test which is comparing the iptables actual and expected output.
> 
> people trust iptables implementation nowadays. as long as you have correct iptable rules, we believe it will behave correctly.

OK. So we should merge this PR as-is to retain the same functionality, then I can later update the test to simply verify the iptables rules are as expected.

File: tests/cacl/test_control_plane_acl.py (hunk 1 above)
Comment (msosyak) [reply tests/cacl/test_control_plane_acl.py]: @jleveque You added `localhost` but forgot to remove `testbed_devices`. Test will fail with 
```
______________________________________________________________________ ERROR at setup of test_control_plane_acls _______________________________________________________________________
file /home/msosyak/bfn-sonic-mgmt/tests/cacl/test_control_plane_acl.py, line 11
  def test_control_plane_acls(duthost, localhost, testbed_devices, creds):
E       fixture 'testbed_devices' not found
```

Comment (jleveque) [reply tests/cacl/test_control_plane_acl.py]: @msosyak: Thanks for the heads-up! PR to remove here: https://github.com/Azure/sonic-mgmt/pull/1689/files

**Pull request merged**